src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Encapsulates the list of tasks user has inputted.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void add(Task t) {
```

Comment:
Suggestion: change t to task so that is clear to read as the code gets longer or gets used in other places